hospital/hospitalfacades/src/org/training/hospital/facades/populators/PathologyPopulator.java

```diff
@@ -19,16 +19,20 @@
  */
 public class PathologyPopulator implements Populator<PathologyModel, PathologyData>
 {
-
-
 	private Converter<PatientModel, PatientData> patientConverter;
+
 	@Override
 	public void populate(final PathologyModel source, final PathologyData target) throws ConversionException
 	{
 
 
 		target.setName(source.getName());
-		target.setPatients(patientConverter.convertAll(source.getPatients()));
+		if(source.getPatients()==null) {
+			target.setPatients(null);
+		}
+		else {
+			target.setPatients(patientConverter.convertAll(source.getPatients()));
+		}
```

Comment:
meglio verificare che la source.getPatients sia != null e fare il set solo in quel caso